src/pages/prob-revshare.astro

```diff
@@ -13,18 +13,23 @@ const description = t("site.description");
     <div class="contentWrapperMd">
       <h1 class="heading2">Probabilistic Revshare Generator</h1>
       <p>Probabilistic revenue sharing (revshare) is one way to share a portion of a web monetized page’s earnings between multiple payment pointers.</p> 
-      <p>Use this tool to define a list of payment pointers and their weights. Then, add the generated monetization <code>link</code> element to your site. The <code>link</code> will contain a unique URL hosted on <code>https://webmonetization.org/api/revshare/pay/</code>. If you'd prefer to not use a hosted URL, you can set up revshare by <a href="/docs/guides/set-up-probabilistic-revenue-sharing" target="_blank">adding a script to your site</a>.</p>
+      <p>Use this tool to define a list of payment pointers and their weights. Then, add the generated monetization <code>link</code> element to your site. The <code>link</code> will contain a unique URL hosted on <code>https://webmonetization.org/api/revshare/pay/</code>. If you'd prefer to not use a hosted URL, you can set up revshare by <a href="/docs/resources/set-up-probabilistic-revenue-sharing" target="_blank">adding a script to your site</a>.</p>
```

Comment:
Surely we need to still move these files if we want the new links to work? They're not working for me, but the files are still in the guides folder.

Reply:
Fixing